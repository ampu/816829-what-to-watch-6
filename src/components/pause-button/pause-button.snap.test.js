import React from 'react';

import {render} from '@testing-library/react';

import PauseButton from './pause-button';

describe(`PauseButton component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<PauseButton/>);
    expect(container).toMatchSnapshot();
  });
});
