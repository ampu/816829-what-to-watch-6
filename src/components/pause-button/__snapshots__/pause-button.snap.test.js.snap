// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PauseButton component Should render according to the snapshot 1`] = `
<div>
  <button
    type="button"
  >
    <svg
      height="21"
      viewBox="0 0 14 21"
      width="14"
    >
      <use
        xlink:href="#pause"
      />
    </svg>
    <span>
      Pause
    </span>
  </button>
</div>
`;
